IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -27,11 +27,13 @@
 #import "MSIDDeviceId.h"
 #import <pthread.h>
 
+static long s_maxQueueSize = 1000;
```

Comment:
Is 1000 a reasonable size? What do other libs use?

Reply:
cocoalumberjack has also 1k

Reply:
I was more wondering about default value. Let's keep it 1k for now and if necessary, we add configuration in future.